### ISSUE TITLE
Strict mode problem

### ISSUE THREAD

**Issue by davedoesdev** [kjur/jsjws#32]
`asn1x509-1.0.js` defines `subjectKeyIdentifier` twice which produces the following warning in Node:

```
Warning: asn1x509-1.0.js:1810
        'subjectKeyIdentifier': '2.5.29.14',
        ^^^^^^^^^^^^^^^^^^^^^^
Duplicate data property in object literal not allowed in strict mode Use --force to continue.
```

Removing one of the occurrences fixes it.

**Comment by kjur** [comment]
Thanks for the report. I'll fix it next update.

**Comment by kjur** [comment]
The 'jsjws' have been merged into jsrsasign 4.8.0 and it was released. Please use this. Regards.

https://github.com/kjur/jsrsasign/
